### PR TITLE
Minor "bug" fixes. (Kidan changes.)

### PR DESCRIPTION
**Pull request by TravellingMerchant**
Badum tish.

Allows Kidan's Antenna to be colour-able in game just like they are currently on the menu.

Also allows Kidan to have underwear and undershirts and adds new suicide messages just for kicks.

No reason not to merge to be honest except the danger of them being too cute. 

Examples:

EDIT: Just to clarify, kidan antenna are already colour-able, this is just fixing it not showing in game.

### CODE REVIEW

**Comment by SamHPurp** [comment]
Colourable antenna look goofy.

**Comment by Superhats** [comment]
Looking goofy is what makes them good. Use cuteness as a natural defense

**Comment by TravellingMerchant** [comment]
So do colour-able vulps and IPC's but we have them.

**Comment by Superhats** [comment]
If we can have toxic green vulps we can have light pink antennas

**Comment by SamHPurp** [comment]
Other things being goofy doesn't excuse it from being goofy.

**Comment by Dinarzad** [comment]
> Other things being goofy doesn't excuse it from being goofy.

[Yeah bugs](https://s-media-cache-ak0.pinimg.com/736x/b7/4b/bd/b74bbd8f6b0dfea784995a27767f3653.jpg) [being brightly colored is weird](http://www.richard-seaman.com/Arthropods/Belize/Highlights/BelizeTigerGrasshopper.jpg) [and doesn't happen at all.](http://www.cubebreaker.com/wp-content/uploads/2014/03/insects-closeup-15.png)
But neon lime green cats an dogs, well I mean that's just normal.

**Comment by taukausanake** [comment]
Do the antennas share the same color with the eyes? Looks kind of weird.

So how long until we get colorized Kidan?

**Comment by Aurorablade** [comment]
It looks off...

**Comment by TravellingMerchant** [comment]
It can be coloured separately from the eyes. It's is own option. I had just made them the same colour in the screen shots for appearances sake. 

What exactly is off about them, @aurorablade?

**Comment by Da-Dman234** [comment]
I can understand the antennae... kinda, but they eyes being bright pink or lime green looks like a joke to me. You can at least somewhat justify disgustingly colored vulps ICly with hair dye (Not that I do) but bug eyes? Literally no reason I can see for them to be outrageous colors other than an arbitrary change someone decided to make that really doesn't make sense. 

Also as SamH said, the solution for vulps being different colors shouldn't be to lower our standard even more with this.

**Comment by TravellingMerchant** [comment]
@da-dman234 kidan eyes are already colourable. This just fixes a bug where the antenna are colourable on the menu and not in game.

**Comment by Xantholne** [comment]
Personally, I think Kidans having more customnization might make their very unpopular race with very little distinctive qualities likely get more people playing them. Especially since they can be unique. 

Many other races already in their own ways have the ability to make themselves as unique as possible; IPC, slime, tangerines, dogbeasts, and so forth. Even Vox now with their colorable tails. I see no reason to allow more colorization to a currently very bland race.

**Comment by TravellingMerchant** [comment]
And to clarify, kidan antenna are already colourable. This is a bug fix making it actually show in game.

**Comment by ShibberinoShipperino** [comment]
I quite like the colourable antennas and such even if it is goofy for neon colours at the end of the day it's up to the guy playin' kidan.

+1 from shipper

**Comment by Lady-Luck** [comment]
lol

**Comment by SheepFighter** [comment]
Can the antennae be made a different color than the eyes? They look rather bleh otherwise.

Regardless, this is bug-fix, so I don't see why it shouldn't be merged.

**Comment by TravellingMerchant** [comment]
Yes, as stated they can be coloured separately.

**Comment by Dinarzad** [comment]
Yo.
Everyone saying "lol colored bugs what? That makes no sense!"
Have you ever fuckin' -seen- a bug before? Here. [Lemme help.](https://tinyurl.com/nxhluh3)
Out of every single animal-based species that exists on paradise the only one that would be acceptable and logical to have neon and bright colored at all, is the one race making people go "BUT MAH COLORS"

Like... what?

**Comment by taukausanake** [comment]
> So how long until we get colorized Kidan?
:^)

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 1]
```diff
@@ -512,6 +512,8 @@
 		"is attempting to bite their antenna off!",
 		"is jamming their claws into their eye sockets!",
 		"is twisting their own neck!",
+		"is cracking their exoskeleton!",
+		"is stabbing themselves with their mandibles!",
```
Comment (Crazylemon64) [inline code/modules/mob/living/carbon/human/species/station.dm]: not explicitly listed but eh whatever

**Pull request merged**